tuf-spec.md

```diff
@@ -1116,7 +1116,7 @@ repo](https://github.com/theupdateframework/specification/issues).
   cycle, report the potential freeze attack.  On the next update cycle, begin
   at step 0 and version N of the root metadata file.
 
-  * **1.9**. **If the timestamp and / or snapshot keys have been rotated, then
+  * **1.9**. **If a threshold of timestamp and / or snapshot keys have been removed, then
```

Comment:
A threshold of timestamp / snapshot keys compared to what? Version N of the root metadata file from Step 1.1?

Reply:
Yes, I will clarify in the text.